contracts/contracts/RequestManager.sol

```diff
@@ -175,7 +175,7 @@ contract RequestManager is Ownable {
         newRequest.targetTokenAddress = targetTokenAddress;
         newRequest.targetAddress = targetAddress;
         newRequest.amount = amount;
-        newRequest.depositReceiver = address(0);
+        newRequest.withdrawInfo = BeamerUtils.FillerInfo(address(0), "0x0");
```

Comment:
I think `bytes32(0)` is clearer.

Reply:
Okay, I wasn't sure what'S the most obvious way.